capa-controller-role/policies/crossplane-policy.json

```diff
@@ -14,7 +14,18 @@
                 "ec2:RevokeSecurityGroupIngress",
                 "cloudwatch:*",
                 "sqs:*",
-                "events:*"
+                "events:*",
+                "iam:GetRole",
+                "iam:CreateRole",
+                "iam:DeleteRole",
+                "iam:TagRole",
+                "iam:GetRolePolicy",
+                "iam:PutRolePolicy",
+                "iam:ListRolePolicies",
```

Comment:
Also `iam:GetPolicyVersion` and `iam:ListPolicyVersions` would be good

Reply:
done